framework/assets/yii.js

```diff
@@ -144,8 +144,8 @@ yii = (function ($) {
          * @param $e the jQuery representation of the element
          */
         handleAction: function ($e, event) {
-            var method = $e.data('method'),
-                $form = $e.closest('form'),
+            var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
```

Comment:
Interestingly, .attr is faster as well. (I did not know or expect that)

https://jsperf.com/jquery-data-vs-attr/22